packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -292,6 +292,7 @@ export const SnapManifestStruct = object({
   initialConnections: optional(InitialConnectionsStruct),
   initialPermissions: PermissionsStruct,
   manifestVersion: literal('0.1'),
+  platformVersion: optional(VersionStruct),
```

Comment:
Hmm, if a Snap adds this before this change is released, their manifest will get rejected 🤔 

Wondering if we should release this first and then add the CLI feature after or something